tests/examples_methods_syntax/diverging_stacked_bar_chart.py

```diff
@@ -262,9 +262,9 @@ def compute_percentages(
 
 y_axis = alt.Axis(title="Question", offset=5, ticks=False, minExtent=60, domain=False)
 
-alt.Chart(source).mark_bar().encode(
+alt.Chart(source).mark_bar().encode( # pyright: ignore[reportArgumentType]
```

Comment:
### Note
- Remove this comment before merging
- It isn't related, would have been needed regardless of overloads